README.md

```diff
@@ -33,6 +33,14 @@ Here's an example
 
 ![image](./docs/img/sendToAnkiOwnDeck.gif)
 
+### Send to dirname deck
+
+Similar to the above but will use the current directory structure as deck name.
+
+Here's an example
```

Comment:
The gif is great but I think it would be nice if there's a bit more text here. 
Such as path/to/file will be come path::to::file as a deck name, using the filename as the title

Reply:
Thanks